guides/node-operators/run-a-super-world-computer-node.md

```diff
@@ -2,89 +2,121 @@
 
 This guide will help you get a Super World Computer node up and running.
 
-## Beta Testnet
+## Software dependencies
```

Comment:
These're not complete dependencies, for complete dependencies, refer to [`mise.toml`](https://github.com/ethstorage/optimism/blob/op-es/mise.toml).